### PR TITLE
Target as symlink

### PR DESCRIPTION
**Pull request by mathstuf**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
-target/
+/target
```
Comment (cramertj) [inline .gitignore]: What is the goal of this change?

Comment (mathstuf) [reply .gitignore]: It applies the change I made to Cargo's gitignore template in rust-lang/cargo#4944.

Comment (cramertj) [reply .gitignore]: What was the motivation for removing the leading slash?

Comment (mathstuf) [reply .gitignore]: Removing? I added the leading one. I can remove it again if wanted. It anchors the ignore to the root of the repo. Not sure why I moved it here. I think I've normally left that alone with these gitignore changes.

Comment (cramertj) [reply .gitignore]: Given that we have `target` inside the nested crates in the workspace as well, I think it makes sense to omit the leading slash.

**Comment by cramertj** [comment]
Thanks!

**Pull request merged**